src/main/java/com/google/api/codegen/transformer/OutputTransformer.java

```diff
@@ -42,13 +43,17 @@
       ImmutableSet.<String>of("response", "response_item");
 
   private final OutputImportTransformer importTransformer;
+  private final PrintArgTransformer printArgTransformer;
```

Comment:
I feel it's a bit unnecessary to have two nested transformer both of which only have one public method. Instead, we can probably move these two methods (`generateOutputImports` and `generatePrintArg`) to `OutputTransformer`, and have each language subclass `OutputTransformer`. Thoughts? @pongad @vchudnov-g